### PR TITLE
AJ-873 -- logic to invoke WDS app creation dependent on Rawls version

### PR DESCRIPTION
**Pull request by aaronkanzer**
We need logic to make sure that only one WDS is deployed when a new workspace is created during migration of app deployment from Terra UI to Rawls.

See https://docs.google.com/document/d/1Y17O_cycv3EsFrce-zFzhpCYPB6I5Xxm3YywSpAr0Q0/edit for the release plan. TL;DR this PR needs review, but should not merge yet.

See https://github.com/broadinstitute/rawls/pull/2283 for the corresponding Rawls PR.

Tested locally.

### CODE REVIEW

**Review by davidangb — COMMENTED** [review]
I think this is a great approach! There's some devil in the details, though:
* we'd want to release this UI change in advance of any Rawls change, so we give enough lead time for the code to make it out to end user's browsers - all those users who don't refresh the page very often
* since we're releasing the UI change in advance … how do we know which Rawls version to set as the `wdsDeploymentRawlsVersion`?

**Review by calypsomatic — APPROVED** [review]

**Review by yuliadub — APPROVED** [review]

File: src/components/NewWorkspaceModal.js [hunk 1]
```diff
@@ -91,6 +92,11 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
     }
   })
 
+  const getRawlsVersion = async () => {
```
Comment (cahrens) [inline src/components/NewWorkspaceModal.js:95]: I am somewhat tempted to just put this in `ajax-common` itself.

**Review by cahrens — APPROVED** [review]

File: src/components/NewWorkspaceModal.js (hunk 1 above)
Comment (davidangb) [reply src/components/NewWorkspaceModal.js:95]: if it informs your opinion, we'll be able to delete this code in a month or so, once we're stably auto-deploying WDS from Rawls.

Comment (cahrens) [reply src/components/NewWorkspaceModal.js:95]: Right, but are you going to delete from both places?

**Pull request merged**